domain-models-runtime/src/test/java/org/folio/rest/persist/PgConnectionMock.java

```diff
@@ -91,6 +96,11 @@ public Future<Transaction> begin() {
     throw new PgConnectionMockException();
   }
 
+  @Override
+  public Transaction transaction() {
+    return null;
+  }
+
```

Comment:
It looks like this PR is mostly about upgrading dependencies. Yet there are these two new method overrides. Is it that one of the dependencies that got upgraded now require these overrides? Seems fine. Just thought I'd ask.

Reply:
I'm not really sure what problem that would solve. I suppose you _could_ do it if you feel it would be an improvement. I see mockito is used elsewhere in RMB.

Reply:
Maybe do a separate PR if you feel like making the change but I think what you have takes care of upgrading the deps.